infrastructure/fargate-demo/template.yaml

```diff
@@ -62,9 +62,9 @@ Resources:
             - !Ref ContainerSecurityGroup
           Subnets:
             - Fn::ImportValue:
-                Fn::Sub: "${CommonStackName}-Subnet1ID"
+                Fn::Sub: "${CommonStackName}-PublicSubnet1ID"
```

Comment:
Ah nice one, was just about to ask about these!

Do we know the fargate deploy still works after this?

Reply:
Yeah I redeployed it at some point to check:

```
fargate-demo (kubernetes) d4ecce2 $ daws sam deploy
Enter token for arn:aws:iam::......

        Deploying with following values
        ===============================
        Stack name                   : hello-node-fargate
        Region                       : eu-west-2
        Confirm changeset            : True
        Disable rollback             : False
        Deployment s3 bucket         : None
        Capabilities                 : ["CAPABILITY_NAMED_IAM"]
        Parameter overrides          : {}
        Signing Profiles             : {}

Initiating deployment
=====================

Waiting for changeset to be created..
Error: No changes to deploy. Stack hello-node-fargate is up to date
```